tests/app/letters/test_letter_utils.py

```diff
@@ -46,38 +43,6 @@ def _sample_precompiled_letter_notification_using_test_key(sample_precompiled_le
     return sample_precompiled_letter_notification
 
 
-@pytest.mark.parametrize(
-    'created_at,folder',
-    [
-        (datetime(2017, 1, 1, 17, 29), '2017-01-01'),
-        (datetime(2017, 1, 1, 17, 31), '2017-01-02'),
-    ],
-)
-@pytest.mark.skip(reason='Letter feature')
-def test_get_bucket_name_and_prefix_for_notification_valid_notification(
```

Comment:
This does not appear to just be a letter feature. Two things are being tested here, one happens to be a letter bucket.

Reply:
@k-macmillan This test is testing the method `get_bucket_name_and_prefix_for_notification` which is only used by which is only used by the method `get_letter_pdf` which is only used by the route `/service/<uuid:service_id>/template/preview/<uuid:notification_id>/<file_type>`

Route `/service/<uuid:service_id>/template/preview/<uuid:notification_id>/<file_type>` is slated for route removal according to our [ VA Notify Endpoints Audit ](https://docs.google.com/spreadsheets/d/18AfqlEOWlrnigauO57jA0VP3RMHTaP4luknqBxrcbuY/edit#gid=115873237)doc

Should this test therefore be removed?

Reply:
This is also a test under for Letter Utilities

Reply:
Yes, should be removed.